src/main/java/com/bart/scorebetlive442/entity/ScoreEntity.java

```diff
@@ -0,0 +1,32 @@
+package com.bart.scorebetlive442.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@Entity
+@Table(name = "Score")
+public class ScoreEntity {
```

Comment:
nazwa tabeli, do przedysktuowania na zajęciach - zobacz że te dane match wyżej (w MatchEntity)  teraz pytanie może tam powinna być relacja do Score ale to trzeba będzie przebudować bo ScoreEntity powinno być raczej tylko id, scoreHome, scoreAway (bez drużyn, ale przy użyciu relacji), bo:

mecz, definiuje że gra drużyna A i B (ale tam masz też info o dacie, więc zakładamy że mecz pomiędzy tymi samymi drużynami, w tym samym układzie gospodarz/gość, ale w innym terminie to inny Match, więc wynik można tam umieścić (jest to charakterystyka tego meczu)